server/src/main/java/org/candlepin/model/CertificateSerialDto.java

```diff
@@ -26,20 +26,20 @@
 @XmlAccessorType(XmlAccessType.PROPERTY)
 public class CertificateSerialDto {
 
-    private Long serial;
+    private String serial;
```

Comment:
@Ceiu  
I didn't find any reference of this class apart from ConsumerResource.getEntitlementCertificateSerials. 

I have changed data type here as part of /consumers/{consumer_uuid}/certificates/serials API response, this method returns the list of serial numbers.

Reply:
@Ceiu 
We can use CertificateSerialDTO instead of CertificateSerialDto but this will change API response format. The API response will have a few extra fields with null values.

Reply:
If the change adds new fields we're okay, even if those fields are null. Just as long as we're not removing or changing data.

Reply:
@Ceiu I have updated ConsumerResource.java to use CertificateSerialDTO instead of CertificateSerialDto. Now API response will be like 

` [ {
  "created" : null,
  "updated" : null,
  "id" : null,
  "serial" : "1838664275955919702",
  "expiration" : null,
  "collected" : null,
  "revoked" : null
} ]`